doc/whats_new/v1.1.rst

```diff
@@ -96,6 +96,12 @@ Changelog
   and :class:`~sklearn.decomposition.TruncatedSVD`. :pr:`21334` by
   `Thomas Fan`_.
 
+:mod:`sklearn.feature_extraction`
+.................................
+
+- |Fix| :class:`feature_extraction.FeatureHasher` now validates input parameters in `transform` instead of `__init__`.
+  :pr:`21573` by :user:`Hannah Bohle <hhnnhh>` and :user:`Maren Westermann <marenwestermann>`.
```

Comment:
To be consistent with the rest of the file, we can wrap the text here:

```rst
- |Fix| :class:`feature_extraction.FeatureHasher` now validates input parameters
  in `transform` instead of `__init__`. :pr:`21573` by
  :user:`Hannah Bohle <hhnnhh>` and :user:`Maren Westermann <marenwestermann>`.
```